tests/unit/session/test_time.py

```diff
@@ -47,6 +46,11 @@ def query_and_wait_mock(query, *args, **kwargs):
 
 
 def test_bqsyncedclock_get_time(bq_client):
+    try:
+        import freezegun
+    except ImportError:
+        pytest.skip("Skip the test when freezegun is not intalled.")
```

Comment:
https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip is a cleaner way to do this.

```suggestion
    freezegun = pytest.importorskip("freezegun")
```